.github/workflows/auto-label.yml

```
@@ -0,0 +1,95 @@
+  contents: read
+  issues: write
+  pull-requests: write
+
+jobs:
+  label-issues:
+    name: Label Issues
+    if: github.event_name == 'issues'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Auto-label issue
+        uses: actions/labeler@v4
```

actionlint (1.7.9)

the runner of "actions/labeler@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)